src/Tools/IdeCoreBenchmarks/SyntaxListBenchmarks.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.IO;
+using BenchmarkDotNet.Attributes;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+
+namespace IdeCoreBenchmarks
+{
+    [MemoryDiagnoser]
+    public class SyntaxListBenchmarks
```

Comment:
>public class SyntaxListBenchmarks [](http://example.com/codeflow?start=4&length=33)

Compiler folks -- let me know if you want me to remove these benchmarks, or if there is a different location you would prefer.

Reply:
If we do check in, it would be good if we didn't hardcode paths at a particular drive/folder.  

Or make it possible to just tweak just that part, and build the rest of the strings from that

Reply:
Changed the path building code to use an existing variable to find the roslyn root.

Reply:
@jjonescz , @cston -- should I go ahead and remove the benchmarks or would you like those merged too?

Reply:
@ToddGrun, it looks like there are a number of existing benchmark tests in src/Tools/Ide*Benchmarks. Those projects appear to be owned by the IDE team, so the decision of whether to include the benchmarks is perhaps up to the IDE team.

Reply:
Removed!